.dev/CRAN_Release.cmd

```diff
@@ -598,22 +599,22 @@ Rdevel CMD check data.table_1.16.0.tar.gz --as-cran  # use latest Rdevel as it m
 #
 bunzip2 inst/tests/*.Rraw.bz2  # decompress *.Rraw again so as not to commit compressed *.Rraw to git
 #
-Resubmit to winbuilder (R-release, R-devel and R-oldrelease)
-Submit to CRAN. Message template :
+# Resubmit to winbuilder (R-release, R-devel and R-oldrelease)
+# Submit to CRAN. Message template :
 ------------------------------------------------------------
 Hello,
 1,016 CRAN revdeps checked. None are impacted.
 Many thanks!
 Best, Matt
 ------------------------------------------------------------
-DO NOT commit or push to GitHub. Leave 4 files (.dev/CRAN_Release.cmd, DESCRIPTION, NEWS and init.c) edited and not committed. Include these in a single and final bump commit below.
-DO NOT even use a PR. Because PRs build binaries and we don't want any binary versions of even release numbers available from anywhere other than CRAN.
-Leave milestone open with a 'release checks' issue open. Keep updating status there.
-** If on EC2, shutdown instance. Otherwise get charged for potentially many days/weeks idle time with no alerts **
-If it's evening, SLEEP.
-It can take a few days for CRAN's checks to run. If any issues arise, backport locally. Resubmit the same even version to CRAN.
-CRAN's first check is automatic and usually received within an hour. WAIT FOR THAT EMAIL.
-When CRAN's email contains "Pretest results OK pending a manual inspection" (or similar), or if not and it is known why not and ok, then bump dev.
+# DO NOT commit or push to GitHub. Leave 4 files (.dev/CRAN_Release.cmd, DESCRIPTION, NEWS and init.c) edited and not committed. Include these in a single and final bump commit below.
+# DO NOT even use a PR. Because PRs build binaries and we don't want any binary versions of even release numbers available from anywhere other than CRAN.
+# Leave milestone open with a 'release checks' issue open. Keep updating status there.
+# ** If on EC2, shutdown instance. Otherwise get charged for potentially many days/weeks idle time with no alerts **
+# If it's evening, SLEEP.
+# It can take a few days for CRAN's checks to run. If any issues arise, backport locally. Resubmit the same even version to CRAN.
+# CRAN's first check is automatic and usually received within an hour. WAIT FOR THAT EMAIL.
+# When CRAN's email contains "Pretest results OK pending a manual inspection" (or similar), or if not and it is known why not and ok, then bump dev.
```

Comment:
why those have been commented out, and lower section _Bump dev for NON-PATCH RELEASE_ stays as is? probably we also the lower section to have comments at start of the lines

Reply:
Agree, added comments for the later sections